src/libraries/wizard/selectcomponentspage.cpp

```diff
@@ -58,11 +61,9 @@ void SelectComponentsPage::initializePage()
 {
     setLibPath();
 
-    // TODO: Tools->Options should allow to configure if default should be metadata or filesystem?
-    m_ui.modeCombo->setCurrentIndex(0);
-    m_ui.modeCombo->setEnabled(LibraryStorage::instance()->library(m_libPath) != nullptr);
-
-    setupModel(FILESYSTEM_COMBO_KEY);
+    const auto lib = LibraryStorage::instance()->library(m_libPath);
+    lib == nullptr ? intializeForSelectedLib(lib, FILESYSTEM_COMBO_IDX, FILESYSTEM_COMBO_KEY)
+                   : intializeForSelectedLib(lib, METADATA_COMBO_IDX, METADATA_COMBO_KEY);
```

Comment:
isnt't there any "find by name" or something in QComboBox?

Reply:
Obviously it is.  I've Amneded commit.